### DISCUSSION TITLE
What do you think about a web interface option for generating a static website?

### DISCUSSION THREAD

**Discussion by nevillepark** [datenstrom/community#895] [Ask a question] [answered]
Hi all, 

One of the things I especially like about yellow is that you can use the web interface for things like adding/editing content and updating the software. 

Currently you can [generate a static site](https://github.com/annaesvensson/yellow-generate) through the command line. What do you think about adding a simple interface on the website so a user could just click a button to generate the static site? Here is a mockup of what it could look like:

![Mockup of Export option in settings menu](https://github.com/datenstrom/yellow/assets/16054979/f32c5c2b-a0e3-4b47-97cb-b02315d92220)

I don't know much about coding, so I'm not sure if it would be easy to add. If it's possible, do you think it would be a good idea? I think it would make it more accessible to users who want a quick way to make a static site and aren't as comfortable with the command line. However, there are no doubt drawbacks I haven't thought about.

**Comment by annaesvensson** [comment, marked as answer]
At the moment no developer is working on this, but would be lovely to have something like this in the future. There are people that don't have SSH access or are not familiar with command line tools, but still want to generate a static website. Maybe someone wants to [make an extension](https://github.com/annaesvensson/yellow-publish)? All that is needed is someone to volunteer. 😄

See also previous discussion datenstrom/community#139.
